ftp.go

```diff
@@ -537,11 +538,15 @@ func (r *Response) Read(buf []byte) (int, error) {
 
 // Close implements the io.Closer interface on a FTP data connection.
```

Comment:
[`io.Closer`](https://godoc.org/io#Closer) doesn't define what happens on multiple invocations, saying that it's up to each implementation to decide and document that:

> The behavior of Close after the first call is undefined. Specific implementations may document their own behavior.

So perhaps it's a good idea to document the new behavior here. Something like:

```
// After the first call, Close will do nothing and return nil.
```

That way, users will know what to expect. Otherwise, they won't know what happens if calling `Close` multiple times without reading source code.